tests/integration/bond_test.py

```diff
@@ -55,6 +56,38 @@ def setup_remove_bond99():
     netapplier.apply(remove_bond)
 
 
+@contextmanager
+def bond_interface(bond_name, slaves_name):
```

Comment:
IMHO the first argument could be just `name` and the second one should be just `slaves`. Since the slaves are a list, the name suffix does not make sense there.

Reply:
And the slaves could be a keyword-argument that defaults to None and is set to an empty list if it is None - or maybe an empty tuple works as well instead of None.

Reply:
Why slaves could be a keyword argument with default value? I think in our case better to leave it as required value, w/o default option

Reply:
The advantage of a keyword argument is that it gives an example for the value. Also it is valid to not specify any slaves by specifying a list, therefore it makes IMHO sense to make it optional.

Reply:
I think we should not allow a bond creation with slaves default = None( except one specific test), almost in all cases bond created with slaves .